pkg/plugin/stream.go

```diff
@@ -39,29 +40,57 @@ func (a *App) SubscribeStream(ctx context.Context, req *backend.SubscribeStreamR
 }
 
 func (a *App) runOpenAIChatCompletionsStream(ctx context.Context, req *backend.RunStreamRequest, sender *backend.StreamSender) error {
-	// Deserialize request data.
-	incomingBody := chatCompletionsRequest{Stream: true}
-	err := json.Unmarshal(req.Data, &incomingBody)
-	if err != nil {
-		return err
-	}
 
-	// Load app settings.
 	settings := loadSettings(*req.PluginContext.AppInstanceSettings)
+	var requestBody map[string]interface{}
```

Comment:
```suggestion
	requestBody := map[string]interface{}{}
```

To avoid 'assignment to nil map' or anything weird like that.